auth-web/src/util/constants.ts

```diff
@@ -147,12 +147,14 @@ export enum InvoiceStatus {
 
 export enum AffiliationTypes {
     NAME_REQUEST ='Name Request',
+    AMALGAMATION = 'Amalgamation Application',
     INCORPORATION_APPLICATION = 'Incorporation Application',
     CORPORATION = 'Corporation',
     REGISTRATION = 'Registration'
 }
 
 export enum FilingTypes {
+    AMALGAMATION = 'amalgamation',
     INCORPORATION_APPLICATION = 'incorporationApplication',
     REGISTRATION = 'registration'
 }
```

Comment:
It sure would be nice to use https://github.com/bcgov/bcrs-shared-components/blob/main/src/enums/filing-types.ts here.

Reply:
LGTM